README.md

```diff
@@ -111,30 +111,15 @@ cp .env.sample .env
 Update `NODE_HOSTNAME` in `.env` to the IP address or hostname of your node. For example:
 
 ```bash
-sed -i -e  s/NODE_HOSTNAME/69.42.690.420/g .env
+sed -i -e s/NODE_HOSTNAME=/NODE_HOSTNAME=69.42.690.420/g .env
 ```
 
 ## A. Deploying a Full Node
 
-### 1. Download Network Genesis
-
-The Poktrolld blockchain deploys various networks: DevNet, TestNet, MainNet, etc...
-
-Access the list of Poktrolld networks available for community participation at [Poktrolld Networks](https://github.com/pokt-network/pocket-network-genesis/tree/master/poktrolld).
-
-Download and place the `genesis.json` for your chosen network (e.g., `testnet-validated`)
-into the `poktrolld/config` directory:
-
-E.g. Testnet-validated:
-
-```bash
-curl https://raw.githubusercontent.com/pokt-network/pocket-network-genesis/master/poktrolld/testnet-validated.json > poktrolld-data/config/genesis.json
-```
-
-### 2. Launch the Node
+### 1. Launch the Node
```

Comment:
Can you make sure the ToC is updated? 

It's just a save operation for me: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one